articles/getting-started-with-browser-sync/index.md

```diff
@@ -0,0 +1,123 @@
+
+```bash
+
+sudo apt install nodejs
+
+```
+
+After the installation, check the version of the installed node package using the command below
+
+```bash
+
+node -v
+
+```
+
+Once you have successfully installed the nodejs package on your computer, let’s get you started on how to install Browser-sync.
+
+```bash
+
+npm install -g browser-sync
+
+```
+
+The above command will get Browser-sync installed on your computer. After that, change the directory to where your project file is located. Looking at the image below, the project file is located on my Desktop. So, I do a ```cd Desktop/Vue project``` in my terminal. This takes me to the Desktop directory of my computer and inside the Vue project folder.
+
+![Change-directory](/engineering-education/getting-started-with-browser-sync/cd.png)
+
+  
+The next thing to do is run the command below:
+
+```bash
+
+browser-sync start --server --directory --files "*"
+
+```
+
+The above command calls for Browser-sync and requests the server to start, then the ```--directory --files``` part tells the server to watch over the files in the directory. Including the "*" means, watch over any changes I make into any file in this directory/folder.
+
+Using Browser-sync for dynamic sites (when you are already running a local server (i.e Xampp) with PHP or a similar scripting language), the process is very similar. Although it will require to run Browser-sync in proxy mode. This can be done using the command below:
+
+```bash
+
+browser-sync start --proxy "localhost/hotel" --files "*"
+
+```
+
+*Where **localhost/hotel** is the URL of your local server.*
+
+Before going to the next step, I will briefly discuss ***servers and proxy servers***
+
+### Server & Proxy Server
+
+_A server_ is a software or hardware device that accepts and responds to requests made over a network.
+
+On the Internet, the term "server" commonly refers to the system that receives requests for web files or data and sends those files back to the client (your computer). Since your project is yet to be hosted on a live server (thus accessible over the internet), your computer serves as a local server that receives requests and fetches your web files.
+
+A _Proxy Server_ acts as the gateway between you and the internet. It’s an intermediary server separating end users from the websites they browse.
+
+If you look back at your previous command, it sounds like this: hey _browser-sync_ please _start_ a _server_ in this _directory_ and make a request for all the _files_ for me. If I make any changes, update it and show me how it looks like :smile:
+
+***Next Step***
+
+The image below has a "\*\*/*\*" instead of the single asterisk "\*" used in the first command, the reason is: after using the first command (browser-sync start --server --directory --files "*") and you switch off your computer or close the terminal, or the working environment and all. To get a complete synchronization and watch over your files again, you have to use “**/*”.
```

Comment:
"and you switch off your computer or close the terminal, or the working environment and all." doesn't make sense. Please revise this to make your point clearer.